### ISSUE TITLE
Uncaught Error: connect EAFNOSUPPORT

### ISSUE THREAD

**Issue by murilomentor** [atom/atom#4957]
After update Atom to version 0.169.0, the error is triggered 19 times everytime I open the editor. It was working perfectly on the previous version.
I suspect the reason is something related to the apm proxy settings on .apmrc (personal info edited):
```
cache=~/.atom/.apm
http-proxy=http://user:password@my.proxy.address:3128
https-proxy=http://user:password@my.proxy.address:3128
strict-ssl=false
```
[Enter steps to reproduce below:]

1. Update Atom to version 0.169.0
2. Open the Atom editor

**Atom Version**: 0.169.0
**System**: Windows 7 SP1
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: connect EAFNOSUPPORT

```
At stream.js:94

Error: connect EAFNOSUPPORT
  at exports._errnoException (util.js:746:11)
  at connect (net.js:840:19)
  at net.js:935:9
  at Object.asyncCallback [as callback] (dns.js:78:16)
  at Object.onlookup [as oncomplete] (dns.js:96:10)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "editor": {
    "fontSize": 12,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
build, v0.22.4
color-picker, v1.2.6
file-icons, v1.4.4
minimap, v3.5.4
project-manager, v1.14.1

# Dev
No dev packages
```

**Comment by kevinsawicki** [comment]
Does this happen when you quit Atom and then relaunch it in safe mode using the `--safe` command line option?

**Comment by murilomentor** [comment]
Yes
Even with the `--safe` argument the errors keep coming up. Apparently they happen when Atom try to update the installed packages, because it also happens when I go into Settings -> Updates or Setttings -> Packages.
I forgot to mention that the program hangs for about 2 minutes until the error notifications are displayed, only then you can use the program.

**Comment by AlexanderMisel** [comment]
**Atom Version**: 0.169.0
**System**: Unknown Windows Version (Windows 8.1 pro in fact)
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: getaddrinfo ENOTFOUND github.com

```
At stream.js:94

Error: getaddrinfo ENOTFOUND github.com
  at errnoException (dns.js:41:10)
  at Object.onlookup [as oncomplete] (dns.js:91:26)

```

### Commands

```
     -1:59.1 core:cut (atom-text-editor.editor)
     -1:55.8 core:paste (atom-text-editor.editor)
     -1:55.0 core:save (atom-text-editor.editor)
  2x -0:00.0 grammar-selector:show (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "editor": {
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
@AlexanderMisel your issue should be fixed if you upgrade to the latest Atom version.

**Comment by AndiDog** [comment]
I also get EAFNOSUPPORT in 0.199 on Windows 7, e.g. when searching packages or loading featured packages. IPv6 is enabled for my network connection (default). No proxy setting. No apmrc changes, just vanilla Atom.

**Comment by kevinsawicki** [comment]
This should be fixed by https://github.com/atom/notifications/pull/62 and available in Atom 1.0.0
